.github/PULL_REQUEST_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,34 @@
+### Feature Overview
+- **Related Issue:** (e.g., Fixes `#<issue_number>`)
+- **Brief Description:**  
+  Summarize the new feature. What problem does it solve? Who benefits from it?
+
+---
+
+### Feature Details
+ - [Redocly docs for this feature/branch](<url>)
+- **New Locations/Nomenclature:**  
+  Specify where new nodes/components have been added (code modules, files, etc.).
+
+- [Dev-Snappi branch reference](<url>)
+- Example command to fetch the dev-snappi branch:
+  ```
+  go get github.com/org/repo@<dev-snappi-branch>
+  ```
+
+---
+
+### Code snippets
+Please provide a concise, well-commented code snippet demonstrating how to use the new feature in gosnappi:
+The snippet should we within
+```go/python
+// please provide good comments for the newly added code snippets it helps in documentation
```

Comment:
'it helps in documentation' is not clear.

Reply:
added more information as to why clean comments are necessary